### DISCUSSION TITLE
How to install codeium?

### DISCUSSION THREAD

**Discussion by aphsa** [NvChad/NvChad#2098] [Q&A] [answered]
I am trying to install the codeium plugin from 

https://github.com/Exafunction/codeium.vim

```lua
 {
    "Exafunction/codeium.vim",
    config = function() end,
  },
```

Mason was able to install the plugin. But I am not able to invoke  ":Codeium Auth"  command

**Comment by siduck** [comment, marked as answer]
@aphsa lazy=false or event="VeryLazy"
